docs/data-sources/drift_detection.md

```diff
@@ -0,0 +1,38 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_drift_detection Data Source - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_drift_detection represents a Drift Detection configuration for a Stack. It will run a proposed run on the given schedule, which you can listen on using run state webhooks. If reconcile is true, then a tracked run will be triggered when drift is detected.
+---
+
+# spacelift_drift_detection (Data Source)
+
+`spacelift_drift_detection` represents a Drift Detection configuration for a Stack. It will run a proposed run on the given schedule, which you can listen on using run state webhooks. If reconcile is true, then a tracked run will be triggered when drift is detected.
```

Comment:
```suggestion
`spacelift_drift_detection` represents a Drift Detection configuration for a Stack. It will trigger a proposed run on the given schedule, which you can listen for using run state webhooks. If reconcile is true, then a tracked run will be triggered when drift is detected.
```